src/libmongoc/src/mongoc/mongoc-openssl.c

```diff
@@ -492,7 +664,7 @@ _contact_ocsp_responder (OCSP_CERTID *id, X509 *peer)
    url_stack = X509_get1_ocsp (peer);
    for (i = 0; i < sk_OPENSSL_STRING_num (url_stack) && !resp; i++) {
       url = sk_OPENSSL_STRING_value (url_stack, i);
-      MONGOC_DEBUG ("Contacting OCSP responder '%s'", url);
+      TRACE ("Contacting OCSP responder '%s'", url);
```

Comment:
I noticed that running example-client would log messages even on success. Debug logs go to stdout by default if no log handler is set. Though that seems like an odd default, I don't see much precedent for using debug logs for successful log messages, just non-critical errors. So I changed these to TRACE logs, which require the driver to be configured with `-DENABLE_TRACE`.